test/models/work_test.rb

```diff
@@ -0,0 +1,207 @@
+require "test_helper"
+
+describe Work do
+  # it "does a thing" do
+  #   value(1+1).must_equal 2
+  # end
+
+  describe 'validations' do
+    before do
+      @work = Work.new(
+          category: "book",
+          title: "test work",
+          creator: "test creator",
+          publication_year: 2003,
+          description: "blah blah"
+      )
+      @work2 = Work.new(
+          category: "movie",
+          title: "test work2",
+          creator: "test creator2",
+          publication_year: 2010,
+          description: "blah blah blah"
+      )
+    end
+
+    it 'is valid when all fields are present' do
+      result = @work.valid?
+
+      expect(result).must_equal true
+    end
+
+    it 'is invalid without a title' do
+      @work.title = nil
+
+      result = @work.valid?
+
+      expect(result).must_equal false
+    end
+
+    it 'is invalid if the title and category exists' do
+
+      existing_work = Work.create(
+          category: "book",
+          title: "test work",
+          creator: "lalala",
+          publication_year: 2010,
+          description: "blah blah"
+      )
+
+      result = @work.valid?
+
+      expect(result).must_equal false
+      expect(@work.errors.messages).must_include :title
```

Comment:
You should also have a check for the error message.  You know it's not valid, but not necessarily why.